src/consts.rs

```diff
@@ -1,5 +1,7 @@
 use fuel_types::bytes::WORD_SIZE;
-use fuel_types::{Bytes32, Salt};
+use fuel_types::{AssetId, Bytes32, Salt};
+
+pub const BASE_ASSET: AssetId = AssetId::zeroed();
```

Comment:
If this is becoming a public constant, then it should be added as associated constant of `AssetId`. ie `AssetId::BASE_ASSET` (or equivalent name)

Reply:
https://github.com/FuelLabs/fuel-types/pull/46